client/landing/jetpack-cloud/sections/scan/history/index.jsx

```diff
@@ -67,9 +68,10 @@ class ScanHistoryPage extends Component {
 		const { value: filter } = this.getCurrentFilter();
 		return (
 			<Main className="history">
-				<QueryJetpackScanHistory siteId={ this.props.siteId } />
 				<DocumentHead title={ translate( 'History' ) } />
 				<SidebarNavigation />
+				<QueryJetpackScanHistory siteId={ this.props.siteId } />
+				<PageViewTracker path="/scan/history/:site" title="History" />
```

Comment:
Should we update it to Scan History?
```suggestion
				<PageViewTracker path="/scan/history/:site" title="Scan History" />
```